app/chat/ls.go

```diff
@@ -535,6 +525,11 @@ func fetchDialogsWithErrorHandling(ctx context.Context, api *tg.Client) ([]dialo
 							ChannelID:  peerType.ChannelID,
 							AccessHash: channel.AccessHash,
 						}
+					} else {
+						// Can't continue pagination without access hash
```

Comment:
First, log.error should be used here. 
Second, the ctx logger in the tdl won't print logs to stdout. Consider using `color.Red`.